ArduCopter/GCS_Mavlink.cpp

```diff
@@ -219,6 +219,25 @@ void NOINLINE Copter::send_rangefinder(mavlink_channel_t chan)
             rangefinder.distance_cm_orient(ROTATION_PITCH_270) * 0.01f,
             rangefinder.voltage_mv_orient(ROTATION_PITCH_270) * 0.001f);
 }
+
+void NOINLINE Copter::send_distance_sensor(mavlink_channel_t chan)
+{
+    for (uint8_t i = 0; i < RANGEFINDER_MAX_INSTANCES; i++) {
```

Comment:
yes, I agree with Francisco that for Copter we should only send the downward facing one for now at least because the send_proximity will send the rest.  The only way this isn't the case is if the proximity library isn't being used.. which is possible but probably rare.